coral/src/app/main.tsx

```diff
@@ -2,11 +2,19 @@ import React from "react";
 import ReactDOM from "react-dom/client";
 import { RouterProvider } from "react-router-dom";
 import router from "src/app/router";
-
 import "@aivenio/design-system/dist/styles.css";
+import { QueryClient, QueryClientProvider } from "@tanstack/react-query";
+import { ReactQueryDevtools } from "@tanstack/react-query-devtools";
+
+// @TODO add vite env
+const DEV_MODE = true;
```

Comment:
[import.meta.env.DEV](https://vitejs.dev/guide/env-and-mode.html#env-variables-and-modes) is the vite way of doing it.